Autocomplete/predefined-items-and-recent-searches/src/predefined-items.js

```diff
@@ -0,0 +1,51 @@
+export const predefinedItems = {
+  getSources({ query }) {
+    if (query) {
+      return [];
+    }
+
+    return [
+      {
+        getSuggestions() {
+          return [
+            {
```

Comment:
If we want to truly make this a reusable plugin, should this array be passed as an option of the plugin?

Reply:
Yes, but this is an example project. So I didn't go there.